### ISSUE TITLE
Proposal: Multi alias/use/import/require consistency check separate for each macro

### ISSUE THREAD

**Issue by Apelsinka223** [rrrene/credo#567]
Hi, thanks for package! I noticed that credo counts multi/single usage of alias/use/import/require macroses to check consistency and I want to propose to count usage separate of each macro. 
For example I used to write aliases with multi syntax and use/import/require with single syntax, and get warnings about use/import/require.

**Comment by TylerWitt** [comment]
I agree. I'm not sure it is correct for a nested module to have the use/import/require modules extracted to the outer module, or anything inside `__using__` definitions of nested modules, though I'm not sure how hard that would be to detect.
